website/data/languages.yml

```diff
@@ -37,3 +37,4 @@
   - "[PHP](https://github.com/prettier/prettier-php)"
   - "[Swift](https://github.com/prettier/prettier-swift)"
   - "[Java](https://github.com/thorbenvh8/prettier-java)"
+  - "[PostgreSQL](https://github.com/benjie/prettier-plugin-pg-sql/)"
```

Comment:
There is also a Ruby one: https://github.com/iamsolankiamit/prettier-ruby, but I'm not sure if we should we put unofficial plugins here.

Reply:
Personally, I'm in favor of it - mainly from the perspective of highlighting that there's interesting things happening and to get people interested to work together. I do see the point of blessing things too early though.